superset/databases/commands/create.py

```diff
@@ -70,13 +71,25 @@ def run(self) -> Model:
         try:
             database = DatabaseDAO.create(self._properties, commit=False)
             database.set_sqlalchemy_uri(database.sqlalchemy_uri)
+            db.session.flush()
+
+            ssh_tunnel = None
+            if ssh_tunnel_properties := self._properties.get("ssh_tunnel"):
+                ssh_tunnel = SSHTunnelDAO.create(
+                    {
+                        **ssh_tunnel_properties,
+                        "database_id": database.id,
+                    },
+                    commit=False,
```

Comment:
what's the difference between creating an instance with the DAO but not committing vs through the model directly like `SSHTunnel(<some properties>)`

Reply:
This will be using the Commands now as shown in the API PR: https://github.com/apache/superset/pull/22199/files#diff-c3f1c5afb1b73f705f01bfa24cb4ca2ac66b84f82875abd3ddb73cb8147ea77eR84 . As for what's the different, I might be wrong but DAO's create is adding the model to the session so we can commit such session later, while just instantiating the model will create the model without it being added to any session so it never persists (unless you go and manually add it to the session I'd assume).

Reply:
got it, cool, thanks.